lib/controller.php

```diff
@@ -185,6 +185,14 @@ public function import_blob($blob) {
 			'_sha'         => $blob->sha,
 		) );
 
+		if(class_exists("WPCom_Markdown")) {
+			$wpcomMd = WPCom_Markdown::get_instance();
+			$args = $wpcomMd -> wp_insert_post_data($args, array(
+				"ID"           => $this->id,
```

Comment:
The controller doesn't have an id property. Also, does the wpcomMd's `wp_insert_post_data` method actually perform the insert?